src/scrabble.js

```diff
@@ -1,5 +1,31 @@
-function scrabble() {
-  // write code here
+function scrabble(scrabbleword) {
+
+  let word = scrabbleword
+  
+  if (word === null) {
+    return 0
+  }
+  if (word === '' || word === " \t\n") {
+    return 0
+  }
+  word = word.toUpperCase();
+  const lettervalues = {
+  'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
+  'D': 2, 'G': 2,
+  'B': 3, 'C': 3, 'M': 3, 'P': 3,
+  'F': 4, 'H': 4, 'V': 4, 'W': 4, 'Y': 4,
+  'K': 5,
+  'J': 8, 'X': 8,
+  'Q': 10,'Z': 10,
+  }
+  let score = 0
+  
```

Comment:
The grouping of different parts of your code that relate to each other so they are close together make it very easy to see what's going on which is great.